src/back-end/products-db.js

```diff
@@ -0,0 +1,132 @@
+import { v4 as uuid } from "uuid";
+import banglesCat from "../asserts/category-images/banglesCat.jpg";
+import earringsCat from "../asserts/category-images/earringsCat.jpg";
+import necklaceCat from "../asserts/category-images/necklaceCat.jpg";
+import pendantCat from "../asserts/category-images/pendantCat.jpg";
+import ringCat from "../asserts/category-images/ringCat.jpg";
+
+
+const categoryList = [
+    {"_id":uuid(),
+    "categoryName":"Rings",
+    "image":ringCat ,
+    },
+    {"_id":uuid(),
+    "categoryName":"Necklace",
+    "image":necklaceCat ,
+    },
+    {"_id":uuid(),
+    "categoryName":"Earrings",
+    "image":earringsCat ,
+    },
+    {"_id":uuid(),
+    "categoryName":"Bangles",
+    "image":banglesCat ,
+    },
+    {"_id":uuid(),
+    "categoryName":"Pendants",
+    "image":pendantCat ,
+    },
+    {"_id":uuid(),
+    "categoryName":"Earrings",
+    "image":earringsCat ,
+    },  
```

Comment:
using ``` " " ``` for key's - keys can only be of string type so you can omit ``` " " ```

Reply:
even though you can use other data types to set an object key, JavaScript automatically converts keys to a string.

Reply:
> even though you can use other data types to set an object key, JavaScript automatically converts keys to a string.

got it Koushith! thank you for explaining